MaaSSim/traveller.py

```diff
@@ -215,7 +215,7 @@ def pax_action(self):
                     # print(self.offers)
                     if len(self.offers) > 1:
                         did_i_opt_out = self.f_platform_choice(traveller=self)
-                    elif len(self.offers) == 1:
+                    elif len(self.offers) == 1 and self.got_offered.triggered:  # TODO: Second condition looks like a hack
```

Comment:
to mi wyglada na jakis hard-coding do bejscia buga - nie wiem czy ten warunek nie wywola niespojnosci potem.

Reply:
Trochę tak, bo jeśli ten warunek byłby niespełniony, to oznacza niespójność danych. Niestety nie udało mi się w sensownym czasie znaleźć innego rozwiązania, aczkolwiek możliwe, że wersja z HEAD mastera repozytorium już ten problem rozwiązała. Sprawdzę to kiedy uda mi się uruchomić cały projekt bez innych błędów.